<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KHotKeys::Window_trigger_widget_ui</class>
  <widget class="QWidget" name="Window_trigger_widget_ui" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>592</width>
        <height>480</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="Q3GroupBox" name="GroupBox4" >
          <property name="title" >
            <string>Trigger When</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QCheckBox" name="window_appears_checkbox" >
                <property name="text" >
                  <string>Window appears</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="window_disappears_checkbox" >
                <property name="text" >
                  <string>Window disappears</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="window_activates_checkbox" >
                <property name="text" >
                  <string>Window activates</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="window_deactivates_checkbox" >
                <property name="text" >
                  <string>Window deactivates</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Q3GroupBox" name="window_groupbox" >
          <property name="title" >
            <string>Window</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="KHotKeys::Windowdef_list_widget" name="windowdef_list_widget" />
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <customwidgets>
    <customwidget>
      <class>KHotKeys::Windowdef_list_widget</class>
      <extends>QWidget</extends>
      <header location="local" >windowdef_list_widget.h</header>
      <sizehint>
        <width>200</width>
        <height>100</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
</ui>
